javascript/javascript3/week2/movies.js

```diff
@@ -0,0 +1,11 @@
+fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+.then(response =>response.json())
+.then(data=>{
+    const badMovies = data.filter(movie=>movie.rating<3);
+    console.log('Number of bad movies:' +badMovies.length);
+})
+.then(data=>{
```

Comment:
the variable _data_ is used twice. This is legal, but not very readable. In this position, you could call it _badMoviesData_